guix_studio/build/vs_2019/msix_package_project/Package.appxmanifest

```diff
@@ -9,12 +9,13 @@
   IgnorableNamespaces="uap rescap">
 
   <Identity
+    Name="Eclipse.EclipseThreadXGUIXStudio"
```

Comment:
We need to leave the name unchanged for now in the identity

Reply:
reverted